src/dmd/mtype.d

```diff
@@ -7353,3 +7353,51 @@ const(char)* toChars(ScopeRef sr) pure nothrow @nogc @safe
         return names[sr];
     }
 }
+
+/** Aliasing happens when there exists another mutable reference to the same
+ * memory object.  The other references can change the contents of the memory
+ * object.  Immutable references may exist, but cannot change the contents. This
+ * function determines if such aliasing of a type `t` is possible.
+ *
+ * Params:
+ *      t = type to check
+ * Returns:
+ *      `true` iff `t` contains any non-`immutable` (mutable or const) indirections.
+ */
+bool hasAliasing(scope Type t)
+{
+    t = t.toBasetype();
+    if (t.isImmutable)
+        return false;
+    else if (auto ts = t.isTypeStruct())
+    {
+        if (auto ti = ts.sym.isInstantiated())
+            if (ti.name == Id.Rebindable &&
```

Comment:
The changelog and function documentation say nothing about Rebindable.

Reply:
Got it, thanks. Let's first see if Rebindable gets vetoed or not. Are for supporting it, @WalterBright?

Reply:
I've removed special handling of Rebindable.